sbin/prepareWorkspace.sh

```diff
@@ -190,11 +190,16 @@ checkoutRequiredCodeToBuild() {
         # shellcheck disable=SC2086
         git fetch --all ${BUILD_CONFIG[SHALLOW_CLONE_OPTION]} || rc=$?
         if [ $rc -eq 0 ]; then
-          git reset --hard "origin/${BUILD_CONFIG[BRANCH]}" || rc=$?
+          git checkout -B "${BUILD_CONFIG[BRANCH]}" "origin/${BUILD_CONFIG[BRANCH]}" || rc=$?
           if [ $rc -eq 0 ]; then
-            echo "Checked out origin/${BUILD_CONFIG[BRANCH]}"
+            git reset --hard "origin/${BUILD_CONFIG[BRANCH]}" || rc=$?
+            if [ $rc -eq 0 ]; then
+              echo "Checked out origin/${BUILD_CONFIG[BRANCH]}"
+            else
+              echo "Failed cmd: git reset --hard \"origin/${BUILD_CONFIG[BRANCH]}\""
+            fi
           else
-            echo "Failed cmd: git reset --hard \"origin/${BUILD_CONFIG[BRANCH]}\""
+            echo "Failed cmd: git checkout -B ${BUILD_CONFIG[BRANCH]} origin/${BUILD_CONFIG[BRANCH]}"
```

Comment:
This should include the quotes for consistency with other failure messages (e.g. lines 199, 208).